### PR TITLE
Update manual page and fix typos in metrics tests

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

**Review by DifferentialOrange — APPROVED** [review]
LGTM about metrics typos

File: doc/man/tarantool.pod [hunk 1]
```diff
@@ -8,43 +8,92 @@ tarantool - a Lua application server and a database management system
...
-formats and more.
+Key features of the application server:
 
-It is designed to let developers quickly create efficient, reliable and
-powerful micro-services and backend applications for the Internet.
+=over
+
+=item *
+
+Heavily optimized Lua interpreter with incredibly fast tracing JIT compiler,
```
Comment (xuniq) [inline doc/man/tarantool.pod:20]:
```suggestion
Heavily optimized Lua interpreter with an incredibly fast tracing JIT compiler
```

File: doc/man/tarantool.pod [hunk 2]
```diff
@@ -8,43 +8,92 @@ tarantool - a Lua application server and a database management system
...
 =over
 
 =item *
 
-Fully compatible with Lua 5.1.
+MessagePack data format and MessagePack based client-server protocol.
+
+=item *
+
+Two data engines: 100% in-memory with complete WAL-based persistence and an own
```
Comment (xuniq) [inline doc/man/tarantool.pod:59]:
```suggestion
Two data engines: a 100% in-memory with complete WAL-based persistence, and an own
```

File: doc/man/tarantool.pod [hunk 3]
```diff
@@ -8,43 +8,92 @@ tarantool - a Lua application server and a database management system
...
+implementation of LSM-tree, to use with large data sets.
```
Comment (xuniq) [inline doc/man/tarantool.pod:60]:
```suggestion
implementation of LSM-tree for the large data sets.
```

File: doc/man/tarantool.pod [hunk 4]
```diff
@@ -71,6 +120,18 @@ Execute the string 'EXPR'.
...
+
+Perform LuaJIT control command.
+
+=item -b ...
+
+Save or list bytecode.
+
+=item -d
+
+Activate debugging session for script.
```
Comment (xuniq) [inline doc/man/tarantool.pod:133]:
```suggestion
Activate debugging session for a script.
```

File: test/metrics-luatest/helper.lua [hunk 5]
```diff
@@ -35,7 +35,7 @@ local function glob_recursive(path)
     return result
 end
 
--- If directory is given then it's scanned recursievly for files
+-- If directory is given then it's scanned recursively for files
 -- ending with `_test.lua`.
 -- If `.lua` file is given then it's used as is.
 -- Resulting list of files is mapped to lua's module names.
```
Comment (xuniq) [inline test/metrics-luatest/helper.lua:41]:
```suggestion
-- If a directory is given, then it's scanned recursively for the files
-- ending with `_test.lua`.
-- If a `.lua` file is given, then it's used as is.
-- The resulting list of files is mapped to Lua's module names.
```

File: test/metrics-luatest/helper.lua [hunk 6]
```diff
@@ -20,7 +20,7 @@ rock_utils.assert_builtin('metrics')
 -- Can't just require luatest internals since the function is not exposed.
 
 -- Returns list of all nested files within given path.
--- As fio.glob does not support `**/*` this method adds `/*` to path and glog it
+-- As fio.glob does not support `**/*` this method adds `/*` to path and glob it
```
Comment (xuniq) [inline test/metrics-luatest/helper.lua:23]:
```suggestion
-- As the `fio.glob()` function does not support `**/*`, this method adds `/*` to the path and globs it
```

File: test/metrics-luatest/helper.lua [hunk 7]
```diff
@@ -20,7 +20,7 @@ rock_utils.assert_builtin('metrics')
 -- Can't just require luatest internals since the function is not exposed.
 
 -- Returns list of all nested files within given path.
```
Comment (xuniq) [inline test/metrics-luatest/helper.lua:22]:
```suggestion
-- Returns a list of all nested files within a given path.
```

**Review by xuniq — APPROVED** [review]
Good, added some suggestions.

**Comment by xuniq** [comment]
Typo in line 15 ([test/metrics-luatest/helper.lua](https://github.com/tarantool/tarantool/pull/8614/files#diff-e248783bda347bb0a8af647653501cd65cbdb227372e162f7d1e0d8c754e980d))

> -- There are may tests in metrics.

may → many ?

File: doc/man/tarantool.pod (hunk 1 above)
Comment (ligurio) [reply doc/man/tarantool.pod:20]: Fixed

File: doc/man/tarantool.pod (hunk 2 above)
Comment (ligurio) [reply doc/man/tarantool.pod:59]: Fixed

File: doc/man/tarantool.pod (hunk 4 above)
Comment (ligurio) [reply doc/man/tarantool.pod:133]: Fixed

File: test/metrics-luatest/helper.lua (hunk 7 above)
Comment (ligurio) [reply test/metrics-luatest/helper.lua:22]: Fixed

File: test/metrics-luatest/helper.lua (hunk 6 above)
Comment (ligurio) [reply test/metrics-luatest/helper.lua:23]: Fixed

File: test/metrics-luatest/helper.lua (hunk 5 above)
Comment (ligurio) [reply test/metrics-luatest/helper.lua:41]: Fixed

**Pull request merged**